### ISSUE TITLE
[Bug]: Crash on inject after update(?)

### ISSUE THREAD

**Issue by okzerus** [Yimura/YimMenu#659]
### Describe the bug

30~ minutes ago my game crashed (as usual) and the rockstar launcher updated itself. (not sure about that, but I got a different loading bar and multiple UAC confirmations)
Now I can't inject the menu into the GTA.

Tried reinstalling the injector, YimMenu, and restarting the PC, but nothing.
The crash log is below.

### Steps To Reproduce

1. Inject the latest dll using Extreme Injector v3
2. GTA will crash in seconds after injecting

### Expected Behavior

things are working

### Screenshots and/or Logs

[17:20:04][INFO/main.cpp:61]: Yim's Menu Initializing
[17:20:04][INFO/main.cpp:62]: Git Info
        Branch: master
        Hash:   675548770c5876ca6dc648f9efdeeb7d90b1cc73
        Date:   Thu Dec 1 18:37:58 2022
[17:20:04][INFO/main.cpp:65]: Settings Loaded.
[17:20:05][DEBUG/module.cpp:49]: Waiting for socialclub.dll...
2022/12/06 17:20:06 330368

***** FATAL EXCEPTION RECEIVED *******

***** Vectored Exception Handler: Received fatal exception EXCEPTION_ACCESS_VIOLATION   PID: 7612

******* STACKDUMP *******

Loaded Modules:

C:\Program Files\NVIDIA Corporation\NvContainer\MessageBus.dll Base Address: 0x7FFEB2C30000 Size: 0x737000
C:\Program Files\Rockstar Games\Social Club\chrome_elf.dll Base Address: 0x7FFEAC4D0000 Size: 0x101000
C:\Program Files\Rockstar Games\Social Club\libcef.dll Base Address: 0x7FFE27850000 Size: 0x809F000
C:\Program Files\Rockstar Games\Social Club\socialclub.dll Base Address: 0x7FFE3F000000 Size: 0xB9B000
C:\ProgramData\Microsoft\Windows Defender\Platform\4.18.2210.6-0\MpOav.dll Base Address: 0x7FFEBB310000 Size: 0x7B000
C:\Users\Admin\AppData\Local\Discord\app-1.0.9007\modules\discord_hook-1\discord_hook\e7081b4d927\DiscordHook64.dll Base Address: 0x7FFE3B500000 Size: 0x127000
C:\Users\Admin\Desktop\Yim\YimMenu.dll Base Address: 0x7FFE3AF00000 Size: 0x239000
C:\WINDOWS\SYSTEM32\AUDIOSES.DLL Base Address: 0x7FFEC0620000 Size: 0x182000
C:\WINDOWS\SYSTEM32\AcGenral.DLL Base Address: 0x7FFEACAE0000 Size: 0x6C000
C:\WINDOWS\SYSTEM32\AcLayers.DLL Base Address: 0x7FFE412D0000 Size: 0x494000
C:\WINDOWS\SYSTEM32\CRYPTBASE.DLL Base Address: 0x7FFEC7460000 Size: 0xC000
C:\WINDOWS\SYSTEM32\CRYPTSP.dll Base Address: 0x7FFEC7440000 Size: 0x18000
C:\WINDOWS\SYSTEM32\CRYPTUI.dll Base Address: 0x7FFEB0BA0000 Size: 0x5E000
C:\WINDOWS\SYSTEM32\CoreMessaging.dll Base Address: 0x7FFEC5060000 Size: 0xF2000
C:\WINDOWS\SYSTEM32\CoreUIComponents.dll Base Address: 0x7FFEC4D00000 Size: 0x35E000
C:\WINDOWS\SYSTEM32\D3D10_1.DLL Base Address: 0x7FFEA0FF0000 Size: 0x31000
C:\WINDOWS\SYSTEM32\D3D11.DLL Base Address: 0x7FFEC3FE0000 Size: 0x263000
C:\WINDOWS\SYSTEM32\D3DCOMPILER_43.dll Base Address: 0x7FFE5E630000 Size: 0x26F000
C:\WINDOWS\SYSTEM32\D3DCompiler_47.dll Base Address: 0x7FFEC4250000 Size: 0x45D000
C:\WINDOWS\SYSTEM32\DINPUT8.dll Base Address: 0x7FFEACA10000 Size: 0x45000
C:\WINDOWS\SYSTEM32\DNSAPI.dll Base Address: 0x7FFEC6F30000 Size: 0xCA000
C:\WINDOWS\SYSTEM32\DSOUND.dll Base Address: 0x7FFE60330000 Size: 0x9C000
C:\WINDOWS\SYSTEM32\DWrite.dll Base Address: 0x7FFEA2BB0000 Size: 0x27F000
C:\WINDOWS\SYSTEM32\ESENT.dll Base Address: 0x7FFEB1460000 Size: 0x32B000
C:\WINDOWS\SYSTEM32\GLU32.dll Base Address: 0x7FFEA1AF0000 Size: 0x2C000
C:\WINDOWS\SYSTEM32\HID.DLL Base Address: 0x7FFEC62F0000 Size: 0xD000
C:\WINDOWS\SYSTEM32\IPHLPAPI.DLL Base Address: 0x7FFEC6EF0000 Size: 0x3B000
C:\WINDOWS\SYSTEM32\MF.dll Base Address: 0x7FFEC2B80000 Size: 0x84000
C:\WINDOWS\SYSTEM32\MFCORE.DLL Base Address: 0x7FFEC26E0000 Size: 0x48E000
C:\WINDOWS\SYSTEM32\MFPlat.DLL Base Address: 0x7FFEC2130000 Size: 0x1BC000
C:\WINDOWS\SYSTEM32\MFReadWrite.dll Base Address: 0x7FFE63A80000 Size: 0x10A000
C:\WINDOWS\SYSTEM32\MPR.dll Base Address: 0x7FFEAE270000 Size: 0x1D000
C:\WINDOWS\SYSTEM32\MSACM32.dll Base Address: 0x7FFE93C20000 Size: 0x1E000
C:\WINDOWS\SYSTEM32\MSASN1.dll Base Address: 0x7FFEC7680000 Size: 0x12000
C:\WINDOWS\SYSTEM32\MSVCP140.dll Base Address: 0x7FFEB2420000 Size: 0x8D000
C:\WINDOWS\SYSTEM32\NETAPI32.dll Base Address: 0x7FFEB89C0000 Size: 0x19000
C:\WINDOWS\SYSTEM32\NETUTILS.DLL Base Address: 0x7FFEC7000000 Size: 0xC000
C:\WINDOWS\SYSTEM32\NTASN1.dll Base Address: 0x7FFEC7520000 Size: 0x3B000
C:\WINDOWS\SYSTEM32\OPENGL32.dll Base Address: 0x7FFEA1030000 Size: 0x125000
C:\WINDOWS\SYSTEM32\PROPSYS.dll Base Address: 0x7FFEC3390000 Size: 0xF6000
C:\WINDOWS\SYSTEM32\RTWorkQ.DLL Base Address: 0x7FFEC1EF0000 Size: 0x34000
C:\WINDOWS\SYSTEM32\SAMCLI.DLL Base Address: 0x7FFEBCEE0000 Size: 0x19000
C:\WINDOWS\SYSTEM32\Secur32.dll Base Address: 0x7FFEB1790000 Size: 0xC000
C:\WINDOWS\SYSTEM32\SspiCli.dll Base Address: 0x7FFEC7950000 Size: 0x32000
C:\WINDOWS\SYSTEM32\UMPDC.dll Base Address: 0x7FFEC6ED0000 Size: 0x12000
C:\WINDOWS\SYSTEM32\USERENV.dll Base Address: 0x7FFEC7990000 Size: 0x2E000
C:\WINDOWS\SYSTEM32\USP10.dll Base Address: 0x7FFE8DAD0000 Size: 0x19000
C:\WINDOWS\SYSTEM32\VCRUNTIME140.dll Base Address: 0x7FFEB8C20000 Size: 0x1B000
C:\WINDOWS\SYSTEM32\VCRUNTIME140_1.dll Base Address: 0x7FFEB8C10000 Size: 0xC000
C:\WINDOWS\SYSTEM32\VERSION.dll Base Address: 0x7FFEC0270000 Size: 0xA000
C:\WINDOWS\SYSTEM32\WINHTTP.dll Base Address: 0x7FFEBB980000 Size: 0x10C000
C:\WINDOWS\SYSTEM32\WININET.dll Base Address: 0x7FFEB82F0000 Size: 0x4D9000
C:\WINDOWS\SYSTEM32\WINMM.dll Base Address: 0x7FFEB43B0000 Size: 0x27000
C:\WINDOWS\SYSTEM32\WINNSI.DLL Base Address: 0x7FFEC0A60000 Size: 0xB000
C:\WINDOWS\SYSTEM32\WINSPOOL.DRV Base Address: 0x7FFEB1ED0000 Size: 0x98000
C:\WINDOWS\SYSTEM32\WINSTA.dll Base Address: 0x7FFEC6790000 Size: 0x64000
C:\WINDOWS\SYSTEM32\WKSCLI.DLL Base Address: 0x7FFEC6C60000 Size: 0x19000
C:\WINDOWS\SYSTEM32\WTSAPI32.dll Base Address: 0x7FFEC35C0000 Size: 0x14000
C:\WINDOWS\SYSTEM32\WindowsCodecs.dll Base Address: 0x7FFEC07B0000 Size: 0x1B4000
C:\WINDOWS\SYSTEM32\Wldp.dll Base Address: 0x7FFEC74F0000 Size: 0x30000
C:\WINDOWS\SYSTEM32\XINPUT1_3.dll Base Address: 0x400000 Size: 0x1E000
C:\WINDOWS\SYSTEM32\XINPUT9_1_0.dll Base Address: 0x7FFEC4770000 Size: 0x7000
C:\WINDOWS\SYSTEM32\amsi.dll Base Address: 0x7FFEBEE60000 Size: 0x1F000
C:\WINDOWS\SYSTEM32\apphelp.dll Base Address: 0x7FFEC5290000 Size: 0x91000
C:\WINDOWS\SYSTEM32\avrt.dll Base Address: 0x7FFEC1EE0000 Size: 0xA000
C:\WINDOWS\SYSTEM32\credui.dll Base Address: 0x7FFE8DAB0000 Size: 0x1A000
C:\WINDOWS\SYSTEM32\cryptnet.dll Base Address: 0x7FFEC0210000 Size: 0x31000
C:\WINDOWS\SYSTEM32\d3d10_1core.dll Base Address: 0x7FFEA4640000 Size: 0xD000
C:\WINDOWS\SYSTEM32\d3d9.dll Base Address: 0x7FFEBAAC0000 Size: 0x1CF000
C:\WINDOWS\SYSTEM32\d3dx9_43.dll Base Address: 0x7FFE3BA40000 Size: 0x263000
C:\WINDOWS\SYSTEM32\dbgcore.DLL Base Address: 0x7FFEB4690000 Size: 0x2C000
C:\WINDOWS\SYSTEM32\dbghelp.dll Base Address: 0x7FFEB4890000 Size: 0x1E4000
C:\WINDOWS\SYSTEM32\devobj.dll Base Address: 0x7FFEC7840000 Size: 0x2C000
C:\WINDOWS\SYSTEM32\dhcpcsvc.DLL Base Address: 0x7FFEC0490000 Size: 0x1D000
C:\WINDOWS\SYSTEM32\dhcpcsvc6.DLL Base Address: 0x7FFEC01F0000 Size: 0x17000
C:\WINDOWS\SYSTEM32\drvstore.dll Base Address: 0x7FFEBB410000 Size: 0x148000
C:\WINDOWS\SYSTEM32\dwmapi.dll Base Address: 0x7FFEC5770000 Size: 0x2F000
C:\WINDOWS\SYSTEM32\dxcore.dll Base Address: 0x7FFEBEC10000 Size: 0x3B000
C:\WINDOWS\SYSTEM32\dxgi.dll Base Address: 0x7FFEC6350000 Size: 0xF3000
C:\WINDOWS\SYSTEM32\dxva2.dll Base Address: 0x7FFE8DAF0000 Size: 0x24000
C:\WINDOWS\SYSTEM32\gpapi.dll Base Address: 0x7FFEC6300000 Size: 0x23000
C:\WINDOWS\SYSTEM32\iertutil.dll Base Address: 0x7FFEB74B0000 Size: 0x2B1000
C:\WINDOWS\SYSTEM32\igd10iumd64.dll Base Address: 0x7FFEBD170000 Size: 0xF5E000
C:\WINDOWS\SYSTEM32\igdusc64.dll Base Address: 0x7FFEBBA90000 Size: 0xA40000
C:\WINDOWS\SYSTEM32\inputhost.dll Base Address: 0x7FFEB3EB0000 Size: 0x152000
C:\WINDOWS\SYSTEM32\kernel.appcore.dll Base Address: 0x7FFEC5950000 Size: 0x12000
C:\WINDOWS\SYSTEM32\ksuser.dll Base Address: 0x7FFEC25E0000 Size: 0x9000
C:\WINDOWS\SYSTEM32\midimap.dll Base Address: 0x7FFE93BB0000 Size: 0xB000
C:\WINDOWS\SYSTEM32\msacm32.drv Base Address: 0x7FFE94560000 Size: 0xD000
C:\WINDOWS\SYSTEM32\msdmo.dll Base Address: 0x7FFEC18F0000 Size: 0xB000
C:\WINDOWS\SYSTEM32\ncrypt.dll Base Address: 0x7FFEC7560000 Size: 0x27000
C:\WINDOWS\SYSTEM32\ntdll.dll Base Address: 0x7FFECA390000 Size: 0x1F8000
C:\WINDOWS\SYSTEM32\ntmarta.dll Base Address: 0x7FFEC6C20000 Size: 0x33000
C:\WINDOWS\SYSTEM32\ondemandconnroutehelper.dll Base Address: 0x7FFE9DDF0000 Size: 0x17000
C:\WINDOWS\SYSTEM32\powrprof.dll Base Address: 0x7FFEC7010000 Size: 0x4B000
C:\WINDOWS\SYSTEM32\profapi.dll Base Address: 0x7FFEC79D0000 Size: 0x1F000
C:\WINDOWS\SYSTEM32\resourcepolicyclient.dll Base Address: 0x7FFEC5560000 Size: 0x14000
C:\WINDOWS\SYSTEM32\sfc.dll Base Address: 0x2531B910000 Size: 0x3000
C:\WINDOWS\SYSTEM32\sfc_os.DLL Base Address: 0x7FFEAE780000 Size: 0x12000
C:\WINDOWS\SYSTEM32\srvcli.dll Base Address: 0x7FFEB7F40000 Size: 0x28000
C:\WINDOWS\SYSTEM32\textinputframework.dll Base Address: 0x7FFEB3C10000 Size: 0xF9000
C:\WINDOWS\SYSTEM32\twinapi.appcore.dll Base Address: 0x7FFEC0AF0000 Size: 0x200000
C:\WINDOWS\SYSTEM32\urlmon.dll Base Address: 0x7FFEB87D0000 Size: 0x1ED000
C:\WINDOWS\SYSTEM32\wdmaud.drv Base Address: 0x7FFE62BB0000 Size: 0x46000
C:\WINDOWS\SYSTEM32\wevtapi.dll Base Address: 0x7FFEC20C0000 Size: 0x65000
C:\WINDOWS\SYSTEM32\windows.storage.dll Base Address: 0x7FFEC5B50000 Size: 0x792000
C:\WINDOWS\SYSTEM32\winmmbase.dll Base Address: 0x7FFE9A320000 Size: 0x26000
C:\WINDOWS\SYSTEM32\wintypes.dll Base Address: 0x7FFEC38C0000 Size: 0x154000
C:\WINDOWS\System32\ADVAPI32.dll Base Address: 0x7FFEC93A0000 Size: 0xAE000
C:\WINDOWS\System32\COMDLG32.dll Base Address: 0x7FFEC9AD0000 Size: 0xDA000
C:\WINDOWS\System32\CRYPT32.dll Base Address: 0x7FFEC7F60000 Size: 0x156000
C:\WINDOWS\System32\DriverStore\FileRepository\nvami.inf_amd64_114eb960cfc3628f\NvCamera\NvCamera64.dll Base Address: 0x7FFE24A90000 Size: 0x866000
C:\WINDOWS\System32\DriverStore\FileRepository\nvami.inf_amd64_114eb960cfc3628f\NvCamera\d3dcompiler_47_64.dll Base Address: 0x7FFE24680000 Size: 0x404000
C:\WINDOWS\System32\DriverStore\FileRepository\nvami.inf_amd64_114eb960cfc3628f\nvldumdx.dll Base Address: 0x7FFE9C260000 Size: 0x106000
C:\WINDOWS\System32\DriverStore\FileRepository\nvami.inf_amd64_114eb960cfc3628f\nvwgf2umx.dll Base Address: 0x7FFE56CE0000 Size: 0x292A000
C:\WINDOWS\System32\GDI32.dll Base Address: 0x7FFEC9970000 Size: 0x2B000
C:\WINDOWS\System32\IMM32.DLL Base Address: 0x7FFEC8A70000 Size: 0x32000
C:\WINDOWS\System32\KERNEL32.DLL Base Address: 0x7FFEC88F0000 Size: 0xBF000
C:\WINDOWS\System32\KERNELBASE.dll Base Address: 0x7FFEC7B30000 Size: 0x2D2000
C:\WINDOWS\System32\MMDevApi.dll Base Address: 0x7FFEC04B0000 Size: 0x85000
C:\WINDOWS\System32\MSCTF.dll Base Address: 0x7FFEC9280000 Size: 0x115000
C:\WINDOWS\System32\NSI.dll Base Address: 0x7FFEC8AB0000 Size: 0x8000
C:\WINDOWS\System32\Normaliz.dll Base Address: 0x7FFEC89B0000 Size: 0x8000
C:\WINDOWS\System32\OLEAUT32.dll Base Address: 0x7FFECA280000 Size: 0xCD000
C:\WINDOWS\System32\PSAPI.DLL Base Address: 0x7FFEC8B20000 Size: 0x8000
C:\WINDOWS\System32\RPCRT4.dll Base Address: 0x7FFEC99A0000 Size: 0x125000
C:\WINDOWS\System32\SETUPAPI.dll Base Address: 0x7FFEC8480000 Size: 0x468000
C:\WINDOWS\System32\SHELL32.dll Base Address: 0x7FFEC8B30000 Size: 0x743000
C:\WINDOWS\System32\SHLWAPI.dll Base Address: 0x7FFEC8AC0000 Size: 0x55000
C:\WINDOWS\System32\USER32.dll Base Address: 0x7FFEC9730000 Size: 0x19D000
C:\WINDOWS\System32\WINTRUST.dll Base Address: 0x7FFEC7A90000 Size: 0x6A000
C:\WINDOWS\System32\WLDAP32.dll Base Address: 0x7FFEC96D0000 Size: 0x56000
C:\WINDOWS\System32\WS2_32.dll Base Address: 0x7FFEC9600000 Size: 0x6B000
C:\WINDOWS\System32\bcrypt.dll Base Address: 0x7FFEC8220000 Size: 0x27000
C:\WINDOWS\System32\bcryptPrimitives.dll Base Address: 0x7FFEC8350000 Size: 0x82000
C:\WINDOWS\System32\cfgmgr32.dll Base Address: 0x7FFEC81D0000 Size: 0x4E000
C:\WINDOWS\System32\clbcatq.dll Base Address: 0x7FFEC9BB0000 Size: 0xAF000
C:\WINDOWS\System32\combase.dll Base Address: 0x7FFEC9C60000 Size: 0x354000
C:\WINDOWS\System32\fwpuclnt.dll Base Address: 0x7FFEBADF0000 Size: 0x80000
C:\WINDOWS\System32\gdi32full.dll Base Address: 0x7FFEC80C0000 Size: 0x10F000
C:\WINDOWS\System32\imagehlp.dll Base Address: 0x7FFEC89C0000 Size: 0x1D000
C:\WINDOWS\System32\msvcp_win.dll Base Address: 0x7FFEC7EC0000 Size: 0x9D000
C:\WINDOWS\System32\msvcrt.dll Base Address: 0x7FFEC83E0000 Size: 0x9E000
C:\WINDOWS\System32\ole32.dll Base Address: 0x7FFEC94D0000 Size: 0x12A000
C:\WINDOWS\System32\sechost.dll Base Address: 0x7FFEC98D0000 Size: 0x9C000
C:\WINDOWS\System32\shcore.dll Base Address: 0x7FFEC9FC0000 Size: 0xAD000
C:\WINDOWS\System32\ucrtbase.dll Base Address: 0x7FFEC8250000 Size: 0x100000
C:\WINDOWS\System32\win32u.dll Base Address: 0x7FFEC7B00000 Size: 0x22000
C:\WINDOWS\System32\winrnr.dll Base Address: 0x7FFEA4450000 Size: 0x12000
C:\WINDOWS\WinSxS\amd64_microsoft.windows.common-controls_6595b64144ccf1df_5.82.19041.1110_none_792d1c772443f647\COMCTL32.dll Base Address: 0x7FFEB1A10000 Size: 0xB0000
C:\WINDOWS\system32\NLAapi.dll Base Address: 0x7FFEC3600000 Size: 0x1D000
C:\WINDOWS\system32\XAudio2_7.dll Base Address: 0x7FFEABDA0000 Size: 0x8B000
C:\WINDOWS\system32\mswsock.dll Base Address: 0x7FFEC7250000 Size: 0x6A000
C:\WINDOWS\system32\napinsp.dll Base Address: 0x7FFEA4490000 Size: 0x17000
C:\WINDOWS\system32\nvapi64.dll Base Address: 0x7FFEB95C0000 Size: 0x761000
C:\WINDOWS\system32\nvspcap64.dll Base Address: 0x7FFE63290000 Size: 0x2C7000
C:\WINDOWS\system32\pnrpnsp.dll Base Address: 0x7FFEA4470000 Size: 0x1B000
C:\WINDOWS\system32\rsaenh.dll Base Address: 0x7FFEC6B00000 Size: 0x34000
C:\WINDOWS\system32\uxtheme.dll Base Address: 0x7FFEC53E0000 Size: 0x9E000
C:\WINDOWS\system32\wshbth.dll Base Address: 0x7FFEA1E40000 Size: 0x15000
C:\Windows\System32\FWPolicyIOMgr.dll Base Address: 0x7FFEB89E0000 Size: 0x51000
C:\Windows\System32\FirewallAPI.dll Base Address: 0x7FFEC6680000 Size: 0xA1000
C:\Windows\System32\OLEACC.dll Base Address: 0x7FFEB1E60000 Size: 0x66000
C:\Windows\System32\WindowManagementAPI.dll Base Address: 0x7FFEC15B0000 Size: 0xA1000
C:\Windows\System32\Windows.UI.dll Base Address: 0x7FFEA7340000 Size: 0x141000
C:\Windows\System32\fwbase.dll Base Address: 0x7FFEC6600000 Size: 0x3A000
C:\Windows\System32\pid.dll Base Address: 0x7FFEC1310000 Size: 0x10000
C:\Windows\System32\rasadhlp.dll Base Address: 0x7FFEBEE10000 Size: 0xA000
C:\Windows\System32\uiautomationcore.dll Base Address: 0x7FFE9AFA0000 Size: 0x2F5000
E:\GTAV\EOSSDK-Win64-Shipping.dll Base Address: 0x7FFE320C0000 Size: 0xA18000
E:\GTAV\GFSDK_ShadowLib.win64.dll Base Address: 0x7FFE3CFB0000 Size: 0x3CC000
E:\GTAV\GFSDK_TXAA_AlphaResolve.win64.dll Base Address: 0x7FFEC2E80000 Size: 0x1C000
E:\GTAV\GTA5.exe Base Address: 0x7FF76D650000 Size: 0x3D20600
E:\GTAV\bink2w64.dll Base Address: 0x7FFE9BB40000 Size: 0x8D000

Dumping registers:

RAX: 0x7FFE3FB9AFF4
RCX: 0x7FFE3F000000
RDX: 0x11
RBX: 0x6
RSI: 0xB9AFEE
RDI: 0x12
RSP: 0xC7B3C4D8D0
RBP: 0xC7B3C4E200
R8:  0x2
R9:  0xB9AFF4
R10: 0x253A1759150
R11: 0x11
R12: 0x2531BA5E478
R13: 0xC7B3C4E3D0
R14: 0xC7B3C4E148
R15: 0x1

stack dump [0]   C:\Users\Admin\Desktop\Yim\YimMenu.dll+0x6A984 0x7FFE3AF6A984
stack dump [1]   C:\Users\Admin\Desktop\Yim\YimMenu.dll+0x699A1 0x7FFE3AF699A1
stack dump [2]   C:\Users\Admin\Desktop\Yim\YimMenu.dll+0x7244C 0x7FFE3AF7244C
stack dump [3]   C:\Users\Admin\Desktop\Yim\YimMenu.dll+0x55AE0 0x7FFE3AF55AE0
stack dump [4]   C:\Users\Admin\Desktop\Yim\YimMenu.dll+0x5A95F 0x7FFE3AF5A95F
stack dump [5]   C:\Users\Admin\Desktop\Yim\YimMenu.dll+0x50CF1 0x7FFE3AF50CF1
stack dump [6]   C:\WINDOWS\System32\KERNEL32.DLL+0x174B4 BaseThreadInitThunk
stack dump [7]   C:\WINDOWS\SYSTEM32\ntdll.dll+0x526A1 RtlUserThreadStart

Exiting after fatal event  (FATAL_EXCEPTION). Fatal type:  EXCEPTION_ACCESS_VIOLATION
Log content flushed successfully to sink

### Store

Epic Games

### Build

- [X] I used YimMenu unmodified

**Comment by Yimura** [comment]
Yes the socialclub.dll updated from the game, this should be fixed after #641 is merged into master.

**Comment by okzerus** [comment]
> Yes the socialclub.dll updated from the game, this should be fixed after #641 is merged into master.

glad to hear! I wonder when should the new update come out? cause this is some serious issue here I think
